### PR TITLE
Update example with correct provider

### PR DESCRIPTION
**Pull request by duncanmurray**
Updating example with new format to include correct provider.

### CODE REVIEW

**Comment by techhat** [comment]
@duncanmurray, you missed a couple. Did you want me to wait on this so you can make them consistent?

**Comment by duncanmurray** [comment]
Updated the ones I missed. Should be all the references covered. Also added the IAD region as well

**Comment by techhat** [comment]
@duncanmurray, not seeing other commits.

File: doc/topics/rackspace.rst [hunk 1]
```diff
@@ -59,14 +59,14 @@ that Salt Cloud has been built to support.
       tenant: 123456
       apikey: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 
-      provider: openstack
+      provider: rackspace
```
Comment (techhat) [inline doc/topics/rackspace.rst]: This is wrong. The rackspace driver will choke on this configuration. The correct driver name for this is openstack.

**Pull request merged**